.gitignore

```diff
@@ -1,4 +1,4 @@
 keys/id_rsa
 keys/id_rsa.pub
 keys.tgz
-
+account_file.json
```

Comment:
I actually forgot to revise it and remove it from .gitignore . It was me avoiding to mistakenly uploading it during local testing. It will be copied and renamed from the secret each time, as per line 106 in .travis (current version). Will correct it.